### DISCUSSION TITLE
Project Update Q1 2024

### DISCUSSION THREAD

**Discussion by DeeEmm** [DIY-Flow-Bench/DIY-Flow-Bench#161] [Announcements]
So it has been some time since I wrote an update for the project. This has been in part as I have been tied up with other projects but mainly as I have been waiting for feedback from testing. Unfortunately, for whatever reason, none of the testers have responded with data, which has caused the project to stall, stuck at the data gathering stage. This is of course a big shame as there is little point in moving forwards until what has already been done is validated.

So in a bid to get the project moving again I will be making another batch of PCBs to expand the beta testing. This will hopefully net some usable results which will allow the project to move on and so be able to finalise the PCB design and code operation. One drawback is that this may potentially result in delaying the final release for another release cycle if further issues are found.

I will put up another post asking for volunteers for the beta test program and try to contact all of those who have expressed an interest. 

I have also (for the time being) decided to reduce the project updates from monthly to quarterly, at least whilst there are no updates to report. This cycle will remain in place until beta testing has finished, at which time we can move back to a regular monthly update and release cycle. 

Cheers

/DM

**Comment by black-top** [comment]
I'm trying to stick to the project, but there's also a lot of other work coming up all the time. 
The main problem at the moment is small vacuum leakage, which affects the measurements. 
I need to rebuild all system and sealt it.

With the code, it seems that everything is fine, but there are still the same problems with wifi that we have talked about several times, but it is not clear what is causing the problems, but it is still possible to use it, so it is no big deal at the moment.

In this case for the data gathering I think the cylinder head flow data would be meaningless.
Maybe it would be useful to make a brief description of the standardized process according to which the data would be measured for compare? 
Currently data capture works without barometric data and without pressure correction, so I think we probably need to write down this data manually for data comparison?  Maybe it would be worth releasing a mini update to save this data automatically?

Other thing that it would probably be best to use the same orifice size or at least similar size for everyone for comparison data? probably the disc saw for orifice?

Another thing, people will probably use different maf in their setups, which complicates data comparison to some extent. 

Maybe it is necessary to decide what the main aims are exactly at this stage.

**Comment by LSRHokie** [comment]
I've been watching this project for a bit, hoping to see more input/feedback/updates on it's progress.  If you need any more beta testers, I'm just sitting by waiting.  I do agree with blacktop though-  I imagine it would be beneficial to limit to a specific maf(s) for now to assist with getting the bugs ironed out?   Nail one down, then expand from there.

**Reply by black-top** [reply]
I think for this stage, step-by-step instructions will be useful. It will be clear to everyone what to do, and it will be easy to compare results. 
For example first step to take all necessary calibrations of flow bench (for example leakage), next step to write what test orifice is used, what style (sharp edge or specific angle ), what diameter etc. Probably the same type and size will be best to use if possible. We need to decide what, may be  standard saw blade? , next step to fill  what weather conditions (barometric pressure, air temperature, humidity), ( automatic data capture will be better, but manual will be ok too), next what test pressure was used, next what flow value was displayed ( corrected and uncorrected) and last what Maf sensor was used, may be additional information like pump type, pipe diameters and etc will be useful to?

**Comment by DeeEmm** [comment]
Sorry for the late reply. I'm currently tied up with renovations so have not been able to get back to this. 

Using a common MAF is a good idea from a data comparison perspective, but the system should produce the same results for any MAF that is used. As with any scientific process the fewer variables that are changed the more controlled (and valid) the comparison. However there is a valid reason for NOT defining these things at this stage.

The initial test was using a commonly available orifice but not defining its characteristics. Similarly not defining the MAF. The expectation is that there will be deviation, but how much, and is this deviation an acceptable amount.

That deviation will come from differences in the orifice, the MAF, the PCB and the bench design. There should be no variation from environmental conditions as that is taken care of with the code and BME. 

If the test shows that the variation generally amounts to 'x' CFM, we can state the accuracy of the project according to this value.  People can then decide is this is accurate enough. It may be 2 cfm, it may be more. But for most people a small inaccuracy would be fine, especially if it is super accessible using only commonly available items (Junk yard MAF sensors and hardware store orifices)

We can then repeat the test with a specified orifice and see how much x reduces by. Ideally this is with a controlled orifice. Then we get a second accuracy value.

Ultimately we should ship around a control MAF and orifice and get everyone to test with it so that all we are then comparing is the bench itself.

I don't know what that variation will be, 2cfm is most likely wishful thinking. Reality is that the variation will likely be much larger and then we need to determine what the cause of this deviation is. Which again may require that we specify the MAF and orifice used or ship a control MAF and orifice.

Given the lack of response I did not want to start off by shipping a MAF and orifice as the likelihood was that I would simply not see it again. I have already done that once.

I do need to get back to this to write some instructions on what tests to carry out, but so far only one person has a working bench and my requests for data from him have fallen on deaf ears.

**Reply by black-top** [reply]
I agree that ship around MAF will be very problematic, expensive and probably, in most setups, not that easy to swap, especially if a different MAF was originally used. I don't think that the same part number MAF but not exact MAF will have big differences in measurements because it should work quote accurately in cars to achieve the right mixtures even with long and short lambda trims. 
Well, for initial testing, any data will be better than nothing. Probably at this stage, the short instructions of the preparation and test what to do and what data to collect will be good enough.
